src/TripSegment/index.js

```diff
@@ -0,0 +1,329 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTokens from "../defaultTokens";
+import CarrierLogo, { StyledCarrierLogo } from "../CarrierLogo";
+import Airplane from "../icons/Airplane";
+import ShowMore from "../icons/ShowMore";
+import ShowLess from "../icons/ShowLess";
+import Bus from "../icons/Bus";
+import Train from "../icons/Train";
+import Text, { StyledText } from "../Text";
+import ToggleUp from "../utils/toggleUp";
+import ToggleDown from "../utils/toggleDown";
+import { CARRIER_TYPE_OPTIONS } from "../CarrierLogo/consts";
+import { getSize } from "../Icon";
+import { ICON_SIZES } from "../Icon/consts";
+
+import type { Props, State } from "./index";
+
+export const Milestone = styled.div`
+  display: flex;
+  height: 50px;
+  width: ${getSize(ICON_SIZES.MEDIUM)};
+  align-items: center;
+  justify-content: space-between;
+  flex: 0 0 auto;
+  cursor: pointer;
+  z-index: 1;
+
+  > svg {
+    z-index: 1;
+    height: ${getSize(ICON_SIZES.MEDIUM)};
+    background: ${({ theme }) => theme.orbit.paletteWhite};
+    transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  }
+`;
+
+Milestone.defaultProps = {
+  theme: defaultTokens,
+};
+
+const arrowStyle = css`
+  position: absolute;
+  content: " ";
+  width: 0;
+  height: 0;
+  border-style: solid;
+  transition: border-color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+`;
+
+arrowStyle.defaultProps = {
+  theme: defaultTokens,
+};
+
+const ArrowDecor = styled.div`
+  position: relative;
+  width: 6px;
+  height: 12px;
+  right: -1px;
+  z-index: 1;
+  &:before {
+    ${arrowStyle};
+    border-width: 6px 6px 6px 0;
+    border-color: transparent ${({ theme }) => theme.orbit.paletteCloudNormal} transparent
+      transparent;
+  }
+  &:after {
+    ${arrowStyle};
+    border-color: transparent ${({ theme }) => theme.orbit.paletteWhite} transparent transparent;
+    border-width: 4px 4px 4px 0;
+    border-color: transparent ${({ theme }) => theme.orbit.paletteWhite} transparent transparent;
+    margin: 2px 0 0 0;
+    right: 0;
+  }
+`;
+
+ArrowDecor.defaultProps = {
+  theme: defaultTokens,
+};
+
+export const StyledSegment = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+  background: ${({ theme }) => theme.orbit.paletteWhite};
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  border: ${({ theme }) =>
+    `${theme.orbit.borderWidthCard} ${theme.orbit.borderStyleCard} ${
+      theme.orbit.paletteCloudNormal
+    }`};
+  transition: border-color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+`;
+
+StyledSegment.defaultProps = {
+  theme: defaultTokens,
+};
+
+const Chevrons = styled.div`
```

Comment:
I am not sure if it looks like in design.

![image](https://user-images.githubusercontent.com/7850592/47736966-1ca7b180-dc70-11e8-8260-3ab6ce5ac6ae.png)

arrows are moved little bit left

Reply:
Yeah, I know. `16px/17px` won't work and you don't want to use decimals, so we should try to fix it afterward, not now.